#
# The message is divided into three parts:
# 1. Header: containing information about general radar parameters
# 2. RawTargets: List of targets containing maximum values in the range-doppler-matrix - used for tracking
# 3. Objects: List of objects generated by the internal tracking algorithm - based on raw targets
Header header
RadarPreHeader radarPreHeader
sensor_msgs/PointCloud2 targets
sick_scan/RadarObject[] objects